neqo-server/src/main.rs

```diff
@@ -90,6 +90,10 @@ struct Args {
     /// Use http 0.9 instead of HTTP/3
     use_old_http: bool,
 
+    #[structopt(name = "max-streams", long, default_value = "16")]
```

Comment:
A point solution here is probably OK, but you only include this one transport parameter and only for the server.  Can you open an issue to expand this further?

Any generic solution should probably include a function to generate connection parameters.